src/libraries/System.Net.Sockets/src/System/Net/Sockets/SafeSocketHandle.cs

```diff
@@ -43,6 +43,8 @@ public SafeSocketHandle(IntPtr preexistingHandle, bool ownsHandle)
 
         internal bool OwnsHandle { get; }
 
+        internal bool PreferInlineCompletions { get; set; } = true;
```

Comment:
I set this to `true` to keep the test simple.
Productized version would set it to `false`.

I can add an envvar to control this. Then we can keep the test, and default to `false`.
The envvar may be useful also for experiments, to be able to control the default for any socket.